### ISSUE TITLE
Signal2vec model and training

### ISSUE THREAD

**Issue by tushara21** [ChristoferNal/multi-nilm#3]
Can see signal2vec using only pretrained encodings in this repo.
Is there a repository for the signal2vec model training and using it with datasets other than the ones in nilmtk ?

**Comment by ChristoferNal** [comment]
Unfortunately there is no code for signal2vec. You ll have to replicate it following the steps at the paper "On time series representations for multi-label NILM" https://link.springer.com/article/10.1007/s00521-020-04916-5
